pkg/ignore/ignore.go

```diff
@@ -51,6 +51,9 @@ func (i *Ignore) Rules(sections ...string) ([]string, error) {
 		}
 		rules = append(rules, slice...)
 	}
+	if len(rules) == 0 {
```

Comment:
Agree with Fran. Being here we might using it accidentally for normal builds for example. In this case, it wouldn't have a real impact. But these charges are for the specific case of remote executions, so I would try to isolate them to that package as Fran suggests

Reply:
Moving it to `createDockerignoreFileWithFilesystem`. My reasoning to add it there was that this function was only used for remotes and given that it's the one that created the rules I added there but I agree that it could be on the other function